dGame/dMission/Mission.cpp

```diff
@@ -421,11 +423,15 @@ void Mission::YieldRewards() {
         }
     }
 
+    __int64 coinsToSend = 0;
```

Comment:
`__int64` is not used anywhere else in the codebase, I believe this can be replaced with `int32_t`.

Reply:
Changed